central/complianceoperator/v2/pipelines/complianceoperatorinfo/pipeline_test.go

```diff
@@ -76,3 +80,104 @@ func (suite *PipelineTestSuite) TestComplianceInfoMsgFromSensor() {
 	}, nil)
 	suite.NoError(err)
 }
+
+func (suite *PipelineTestSuite) TestComplianceInfoMinimalRequiredVersion() {
+	minVersion := env.ComplianceMinimalSupportedVersion.VersionSetting()
```

Comment:
(optional) This is a test, so hardcoding the `v1.6.0` here makes sense. With that, we can prevent that other CI test runs define or change that setting and leave it uncleaned.

Reply:
I'll use `env.ComplianceMinimalSupportedVersion.DefaultValue()` - if that is ok?

Reply:
Yes, that would also do the job.